tests/python/unittest/test_runtime_micro.py

```diff
@@ -25,8 +25,25 @@
 from tvm.micro import create_micro_mod
 from tvm.relay.testing import resnet
 
-# Use the host emulated micro device.
-DEV_CONFIG = micro.device.host.default_config()
+# # Use the host emulated micro device.
+DEV_CONFIG_A = micro.device.host.generate_config()
+DEV_CONFIG_B = micro.device.host.generate_config()
+TARGET = 'c -device=micro_dev'
```

Comment:
these can be re-collapsed into a single `DEV_CONFIG`. I separated them for prototyping, because you need separate server ports for physical devices

Reply:
we still have test_interleave_sessions for now though?